company-kata-solutions/src/test/java/org/eclipse/collections/companykata/Exercise4Test.java

```diff
@@ -94,9 +98,9 @@ public void whoSuppliesSandwichToaster()
     @Test
     public void filterOrderValues()
     {
-        List<Order> orders = this.company.getMostRecentCustomer().getOrders();
-        MutableList<Double> orderValues = null;
-        MutableList<Double> filtered = null;
+
+        MutableList<Double> filtered = ListIterate.collect(
```

Comment:
Please break this up into two statements with both `orderValues` and `filtered` variables as the test has.